packages/fiori/src/themes/ViewSettingsDialog.css

```diff
@@ -54,3 +72,18 @@
 	overflow: hidden;
 }
 
```

Comment:
I think that there is a style in the bar which should be overridden here (or is it wrong in the bar?)
::slotted(.ui5-bar-content) {
	margin: 0 0.25rem;
}
---> the left margin seems to be not needed.

Reply:
is this fixed?